[TASK] Add note about breaking flex form section change (#928)
TYPO3-Documentation/TYPO3CMS-Reference-TCA: Documentation/ColumnsConfig/Type/Flex/FlexformSyntax.rst

-  Charset follows that of the current backend UTF-8. When storing FlexForm information in external files,
   make sure that they are using UTF-8 too.

> -  :php:`type='inline'` and other types that point to different tables are not allowed in flex form section containers.

Reviewer: ```suggestion
-  :php:`type='inline'` and other types that point to different tables are not allowed in FlexForm section containers.
```

.. versionchanged:: 13.0

    Since TYPO3 13.0, also :php:`type='select'` (when using

>     :php`foreign_table`) is not allowed to be used and will raise an exception

Reviewer 2: looks like one `:` is missing here after php

Reviewer: Thanks, fixed it.

>     :php`foreign_table`) is not allowed to be used and will raise an exception

Reviewer: ```suggestion
    :php:`foreign_table`) is not allowed to be used and will raise an exception
```

>     :php:`foreign_table`) is not allowed to be used and will raise an exception

Reviewer: ```suggestion
    :php:`foreign_table`) is not allowed and will raise an exception
```

Omit two times "used".

>     when used. Note this only applies to flex form sections, not general

Reviewer: ```suggestion
    when used. Note this only applies to FlexForm sections, not general
```

>     flexform usage.

Reviewer: ```suggestion
    FlexForm usage.
```
